addons/crm/tests/test_crm_lead_merge.py

```diff
@@ -294,3 +390,120 @@ def test_merge_method_dependencies(self):
         # 2many accessors updated
         self.assertEqual(master_lead.activity_ids, activity)
         self.assertEqual(master_lead.calendar_event_ids, calendar_event)
+
+    @users('user_sales_manager')
+    @mute_logger('odoo.models.unlink')
+    def test_merge_method_followers(self):
+        """ Test that the followers of the leads are added in the destination lead.
+
+        They should be added if:
+        - The related partner was active on the lead (posted a message in the last 7 days)
+        - The related partner is not already following the destination lead
+        """
+        self.env['crm.lead'].browse(self.lead_w_partner_company.ids).write({'type': 'opportunity'})
```

Comment:
Why this change ? Can't we rely on standard test data ? We already have some tests about confidence computation normally. That way we keep scope of the test only for followers.